Source/CarthageKit/Resolver.swift

```diff
@@ -163,9 +164,10 @@ public struct Resolver {
 	/// the specified node (or as a root otherwise).
 	///
 	/// This is a helper method, and not meant to be called from outside.
-	private func graphs(for nodes: [DependencyNode], dependencyOf: DependencyNode?, basedOnGraph inputGraph: DependencyGraph) -> SignalProducer<DependencyGraph, CarthageError> {
+	private func graphs(for nodes: [DependencyNode], dependencyOf: DependencyNode?,
+	                    basedOnGraph inputGraph: DependencyGraph) -> SignalProducer<DependencyGraph, CarthageError> {
```

Comment:
Makes sense then. I just wonder why I got a lot of `vertical_parameter_alignment` warnings from SwiftLint then ... are they not aware of this style? I've found many places where it had the wrong style, I've updated them all. This should be fixed now, too.